hpx/runtime/threads/policies/lockfree_queue_backends.hpp

```diff
@@ -17,8 +17,12 @@
 #include <boost/lockfree/queue.hpp>
 #endif
 
+// Does not rely on CXX11_STD_ATOMIC_128BIT
```

Comment:
Is this comment a left-over or is it still true? If yes, please guard all of the code using 128Bit atomics (using `HPX_HAVE_CXX11_STD_ATOMIC_128BIT`).

Reply:
Sorry, I misread the comment, please disregard.

Reply:
if the moody camel code used 128 bit atomics internally, would there be a simple way of checking (ie compiling with a certain compiler flag and triggering an error?)

Reply:
The `moodycamel::ConcurrentQueue` uses atomics of the `size_t` and `index_t` types from its traits, which are `std::size_t` by default. There's no use case for 128-bit atomics in the queue.

Reply:
@cameron314 thanks for clarifying!